mne/epochs.py

```diff
@@ -1511,6 +1511,9 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             a str (single event) or a list of str. In the case where one of
             the entries is a list of str, event_ids in that list will be
             grouped together before equalizing trial counts across conditions.
+            In the case where partial matching (using event_ids with '/') is
+            used, processing works as if the event_ids matched by the provided
+            tags had been supplied instead.
```

Comment:
Perhaps rephrase for clarity? e.g.
" In the case where partial matching is used  (using '/' in `event_ids`), `event_ids` will be matched according to the provided tags"